wiki/Submission/Beatmap_Ranking_Procedure/en.md

```diff
@@ -1,44 +1,23 @@
+1.  The beatmapset meets the [Ranking Criteria](/wiki/Ranking_Criteria).
+2.  The beatmapset meets overall quality standards that are not part of the Ranking Criteria.
 
-### Mode-Specific Bubbles
+## Nomination Reset
 
-Mode-Specific Bubbles are bubbles applied to hybrid mapsets for specific modes. The criteria for any of these bubbles is regulated to its respective [Ranking Criteria](/wiki/Ranking_Criteria), however only Mode-Specific BNs can apply these bubbles. For a list of available BNs, see [the respective list on its article](/wiki/People/Beatmap_Nomination_Group).
+Updating a nominated beatmapset will cause the nomination count to be reset. This means it will need to be nominated again. Alternatively, a nomination can be reset by another BN or QAT, if either the beatmapset does not follow one or more rules in the Ranking Criteria or the BN has issues with the quality of the map. If a beatmapset is reset due to quality standards not part of the Ranking Criteria, then it is called a veto. Vetoes are dealt with mediation by the QAT. 
 
-Popped Bubble ![Bubble pop icon](/wiki/shared/icon/bubble-pop.gif "Bubble pop icon")
----------------------------------------------------------------------------------
+Note: In the case of a minor change to a nominated beatmapset, then the BN can nominate the beatmapset again immediately, but in the case of a veto, a decision from the QAT needs to be done before the map is allowed to be nominated again.
 
-Updating a Bubbled beatmap will cause this icon to replace it, it means it needs to get into bubbled status again, alternatively a BN that disagrees with the judgment of the other staff member may place this icon as well. Some BNs place this icon on your map while modding it to prevent another BN to look at it, and is very likely it will get ranked afterwards.
+## Qualified and Ranked Sections
 
-Note: If your map has a bubble, then it will be popped if you update it. If the update was not drastic (e.g. All you did was fix a [slider](/wiki/Hit_Objects#sliders)), then just ask the BN who bubbled it to rebubble it. Your map will gain 5 [SP](/wiki/Glossary/#star-priority) for a popped bubble icon.
+Once a beatmapset reaches two nominations, it will reach the Qualified section. While in the Qualified section, the map will gain a leaderboard, but will not award [Performance Points](/wiki/Performance_Points). The reason two Beatmap Nominators are required to check a beatmapset is because human beings make mistakes, and this greatly reduces the likelihood of a map making it through the ranking system with a drastic error. If a beatmap gets qualified but an unrankable issue is found or improvements to the beatmap have been proposed and accepted, then the beatmapset will be disqualified by a QAT member. This resets the nominations on the map. 
```

Comment:
```suggestion
Once a beatmapset reaches two nominations, it will reach the Qualified category. While in the Qualified category, the map will gain a leaderboard, but will not award [Performance Points](/wiki/Performance_Points). The reason two Beatmap Nominators are required to check a beatmapset is because human beings make mistakes, and this greatly reduces the likelihood of a map making it through the ranking system with a drastic error. If a beatmap gets qualified but an unrankable issue is found or improvements to the beatmap have been proposed and accepted, then the beatmapset will be disqualified by a QAT member. This resets the nominations on the map. 
```